### PR TITLE
unhardcode API_HOST in nginx

### LINKED ISSUE #114: docker: nginx: hardcoded `api:8000`
`api` isn't available, `spleeterapi` is used instead.

### PR DESCRIPTION
**Pull request by jtagcat**
Closes #114

### CODE REVIEW

**Comment by JeffreyCA** [comment]
Hi, thanks for adding this. Can you add the same to docker-compose.yml and the README?

**Comment by jtagcat** [comment]
Done, also added default in the nginx Dockerfile.

File: README.md [hunk 1]
```diff
@@ -205,7 +205,8 @@ Here is a list of all the environment variables you can use to further customize
 |---|---|
 | `CPU_SEPARATION` | No need to set this if using Docker. Otherwise, set to `1` if you want CPU separation and `0` if you want GPU separation.
 | `DJANGO_DEVELOPMENT` | Set to `true` if you want to run development build, which uses `settings_dev.py`/`settings_docker_dev.py` and runs Webpack in dev mode. |
-| `APP_HOST` | Domain name or public IP of server. This is only used for production builds (i.e. when `DJANGO_DEVELOPMENT` is not set) |
+| `APP_HOST` | Domain name or public IP of server. This is only used for production builds (i.e. when `DJANGO_DEVELOPMENT` is not set)
```
Comment (JeffreyCA) [inline README.md:208]:
```suggestion
| `APP_HOST` | Domain name or public IP of server. This is only used for production builds (i.e. when `DJANGO_DEVELOPMENT` is not set) |
```

File: README.md [hunk 2]
```diff
@@ -205,7 +205,8 @@ Here is a list of all the environment variables you can use to further customize
 |---|---|
 | `CPU_SEPARATION` | No need to set this if using Docker. Otherwise, set to `1` if you want CPU separation and `0` if you want GPU separation.
 | `DJANGO_DEVELOPMENT` | Set to `true` if you want to run development build, which uses `settings_dev.py`/`settings_docker_dev.py` and runs Webpack in dev mode. |
-| `APP_HOST` | Domain name or public IP of server. This is only used for production builds (i.e. when `DJANGO_DEVELOPMENT` is not set) |
+| `APP_HOST` | Domain name or public IP of server. This is only used for production builds (i.e. when `DJANGO_DEVELOPMENT` is not set) 
+| `API_HOST` | Hostname of API server (for nginx)
```
Comment (JeffreyCA) [inline README.md:209]:
```suggestion
| `API_HOST` | Hostname of API server (for nginx) |
```

**Review by JeffreyCA — CHANGES_REQUESTED** [review]
Great thank you! I believe you're the first community contributor 😊

**Comment by jtagcat** [comment]
If you start counting from zero (you), then yes.

**Pull request merged**